src/jokes.ts

```diff
@@ -60,6 +60,43 @@ class JokesLoader {
     }
   }
 
+  public async deleteJoke(joke: Joke): Promise<{ success: boolean; joke_id?: number; error?: string }> {
+    const jokesPath = path.join(__dirname, '../blagues.json');
+    const req = await this.checkAccess(jokesPath);
+
+    if (!req.success) return req;
+
+    try {
+      await this.loader.wait();
+
+      const rawData = await fs.readFile(jokesPath, 'utf-8');
+      const jokes = (rawData.length ? JSON.parse(rawData) : []) as Joke[];
+      const toDelJoke = jokes.find((j) => j.id === joke.id);
```

Comment:
```suggestion
      const jokeToDelete = jokes.find((j) => j.id === joke.id);
```
Je pense que rénover la constante de cette manière serais plus clair, et correspondrait plus aux conventions respectes jusque là sur le projet je trouve 🤔

Reply:
C'est vrai, à la base c'était une variable provisoire pour test, mais j'ai oublié de la renommer 😅